Makefile

```diff
@@ -71,7 +70,7 @@ $(WINDOWS_AMD64_BINARY): $(SOURCES) GITCOMMIT_SHA
 	./scripts/build_variant.sh windows amd64 $(VERSION) $(shell cat GITCOMMIT_SHA)
 	@mv ./bin/windows-amd64/$(BINARY_NAME) ./$(WINDOWS_AMD64_BINARY)
 
-GITCOMMIT_SHA: $(GITFILES)
```

Comment:
Can you revert this change?  `$(GITFILES)` expresses a dependency in make so that the `GITCOMMIT_SHA` file gets regenerated any time there is a change to the files inside `.git`.  This ensures that `GITCOMMIT_SHA` gets updated when you make new commits, when you pull changes from a remote, and when you switch branches.

Reply:
Ah, that makes sense. Thanks for the explanation!

Could we not instead make this a `.PHONY` target and remove the `$(GITFILES) dependency? It would mean that it would run on _every_ `make`, but OTOH we're already checking everything in the `.git/` dir for changes on every `make` invocation, right?

Reply:
Making the `GITCOMMIT_SHA` target run on every `make` would mean that we'd end up rebuilding the binaries on each `make` invocation as well (since `GITCOMMIT_SHA` is a dependency of the binary targets).